domains/authorization/Authorization.API/API/Authorization/[Features]/GrantConsentToClientCommand.cs

```diff
@@ -65,7 +65,7 @@ public async Task Handle(GrantConsentCommand command, CancellationToken cancella
     }
 }
 
-public record GrantConsentCommand(Guid UserId, string OrganizationCvr, IdpClientId IdpClientId) : IRequest;
+public record GrantConsentToClientCommand(Guid UserId, string OrganizationCvr, IdpClientId IdpClientId) : IRequest;
 
-public class UserNotAffiliatedWithOrganizationCommandException()
-    : ForbiddenException("Not authorized to perform action");
+public class UserNotAffiliatedWithOrganizationCommandException() : ForbiddenException("Not authorized to perform action");
+public class UnableToGrantConsentToOwnOrganization() : ForbiddenException("Not able to grant consent to own organization");
```

Comment:
Jeg troede at api'et returnerede 404 eller 204, hvis man er unauthorized. hvorfor skal man så have denne her?

Reply:
det skal ikke være 404?

Reply:
No, i believe attempting to grant consent to you own organization is a BAD request.